bindings/c/local.mk

```diff
@@ -22,8 +22,8 @@
 
 fdb_c_CFLAGS := $(fdbclient_CFLAGS)
 fdb_c_LDFLAGS := $(fdbrpc_LDFLAGS)
-fdb_c_LIBS := lib/libfdbclient.a lib/libfdbrpc.a lib/libflow.a $(TLS_LIBS)
-fdb_c_STATIC_LIBS :=
+fdb_c_LIBS := lib/libfdbclient.a lib/libfdbrpc.a lib/libflow.a
+fdb_c_STATIC_LIBS := $(TLS_LIBS)
```

Comment:
Yes and no.  For this specific problem, I'd prefer to have (wrongly) the linux and macOS build instructions stay the same.  But tangentially yes, because having a working LDFLAGS override feels like a correct thing to do anyway, and I'll see if I can slide that into a future commit.